deploy/olm-catalog/jaeger-operator/1.14.0/jaeger-operator.v1.14.0.clusterserviceversion.yaml

```diff
@@ -94,19 +94,19 @@ spec:
     * Service dependency analysis
     * Performance / latency optimization
     * OpenTracing compatible data model
-    * Multiple storage backends: Cassandra, Elasticsearch, memory.
+    * Multiple storage backends: Cassandra, Elasticsearch, Memory, Badger.
 
     ### Operator features
 
-    * **Multiple modes** - Supports `allInOne`, `production`, and `streaming` [modes of deployment](https://github.com/jaegertracing/jaeger-operator#strategies).
+    * **Multiple modes** - Supports `allInOne`, `production`, and `streaming` [modes of deployment](https://www.jaegertracing.io/docs/latest/operator/#deployment-strategies).
```

Comment:
Are the "modes" here deployment strategies or Helm charts or something else?  (Wondering if "production" should be "production-elasticsearch"? And if that is "allInOne" or "all-in-one"?)

Reply:
It is actually the 'strategy' (architecture) used - independent of the storage which is specified separately.
It is `allInOne`.
This is different to the template used in the maistra operator's CR, which is a combination of the strategy and storage choice.

Reply:
Different upstream/downstream might explain why I keep getting confused!